### DISCUSSION TITLE
HTML upload to OBJKT/VERSUM not loading

### DISCUSSION THREAD

**Discussion by micahalhadeff** [cables-gl/cables_docs#309] [Answers & Questions] [answered]
Hello,

I am trying to mint my Cables project on Objkt or Versum and when I am previewing it on the minting page it looks fine and works but once it's minted all I get is a black screen.. Even after waiting hours and hours for it to cache. I thought it might have to do with using Draco compression with the project but I uploaded it without that feature and it still just loads a black screen... any ideas? 

here is a link to the project:
https://versum.xyz/token/versum/28207

Thanks!

**Comment by KOLEKTYW-online** [comment, marked as answer]
Hi, I do it like this:
1. Move patch.js and assets to the main folder, next to index.html.
2. Change file paths. In index.html change:
<script type="text/javascript" src="js/patch.js" async></script>
to
<script type="text/javascript" src="patch.js" async></script>
3. In patch.js find all the asset paths and remove "assets/" from paths.
4. In index.html add <meta property=”og:image” content=”thumbnail.jpeg”/> (of course file can have a different name)

**Reply by micahalhadeff** [reply]
awesome!! thank you! the 4th step "in index.html add" what am i adding? sorry if these questions sound obvious

**Reply by micahalhadeff** [reply]
update! i got it to work! i followed your directions exactly and i got it to load. thank you so much!!!

**Reply by KOLEKTYW-online** [reply]
Sorry, didn't paste the code.
<meta property=”og:image” content=”thumbnail.jpg”/>

**Reply by steam0r** [reply]
`<meta property=”og:image” content=”thumbnail.jpg”/>`

**Reply by steam0r** [reply]
actually the meta og:image needs to be an absolute url, including domain, so this will not work (but should not keep your patch from working)
